Sources/Screens/SignOutConfirmation/SignOutViewModel.swift

```diff
@@ -0,0 +1,62 @@
+import GDSCommon
+import Logging
+import UIKit
+
+struct SignOutPageViewModel: GDSInstructionsViewModel, BaseViewModel {
+    let title: GDSLocalisedString = "app_signOutConfirmationTitle"
+    let body: String = GDSLocalisedString(stringLiteral: "app_signOutConfirmationBody1").value
+    var childView = UIView()
+    let buttonViewModel: ButtonViewModel
+    let rightBarButtonTitle: GDSLocalisedString? = "app_cancelButton"
+    
+    let secondaryButtonViewModel: (any ButtonViewModel)? = nil
+    let backButtonIsHidden: Bool = true
+    let analyticsService: AnalyticsService
+
+    init(analyticsService: AnalyticsService,
+         buttonAction: @escaping () -> Void) {
+        self.analyticsService = analyticsService
+        self.buttonViewModel = AnalyticsButtonViewModel(titleKey: "app_signOutAndDeleteAppDataButton",
+                                                        backgroundColor: .gdsRed,
+                                                        analyticsService: analyticsService) {
+            buttonAction()
+        }
+        self.childView = configureStackView()
+
+    }
+
+    func didAppear() {
+
+    }
+
+    func didDismiss() {
+
+    }
+
+    private func configureStackView() -> UIView {
+        let body2Label = UILabel()
+        body2Label.accessibilityIdentifier = "sign-out-body2-text"
```

Comment:
can you please move the setting of the accessibilityIdentifiers down to where other attributes of the label are being set